blocks/image-compare/src/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Image Compare Block
+ * A block that allows inserting side-by-side image comparison.
+ */
+
+
+// WordPress dependencies
+import { registerBlockType } from '@wordpress/blocks';
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export function registerBlock() {
+
+	registerBlockType( 'jetpack/image-compare-block', {
+
+		title: 'Image Compare',
+
+		icon: 'image-flip-horizontal',
```

Comment:
Let's use this icon:
<img width="405" alt="Screenshot 2020-03-18 at 09 46 31" src="https://user-images.githubusercontent.com/1204802/76942468-131ef800-68fe-11ea-8614-3f1f3df49c44.png">

````
<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
	<Path d="M3 5v14c0 1.1.9 2 2 2h8.5V3H5c-1.1 0-2 .9-2 2zm9 14.5H5c-.3 0-.5-.2-.5-.5v-2.4l4.1-3 3 1.9c.1.1.3.1.4.1v3.9zm0-5.7L9 12c-.1-.1-.3-.1-.4-.1-.2 0-.3 0-.4.1l-3.6 2.6V5c0-.3.2-.5.5-.5h7v9.3zM19 3h-4v1.5h4c.3 0 .5.2.5.5v14c0 .3-.2.5-.5.5h-4V21h4c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z" />
</SVG>
```

Remember to import `SVG` and `Path` components so it works in React Native.

I believe you can just drop the SVG right inline here.

Reply:
Added.